wondrous-app/components/profile/modals/index.tsx

```diff
@@ -105,12 +126,70 @@ export const MoreInfoModal = (props) => {
     fetchPolicy: 'cache-and-network',
   });
   const pods = orgPodData?.getOrgPods;
+
+  const searchUser = (value) => {
+    if (value) {
+      const newList = userList.filter((item) => {
+        if (item.username && item.bio) {
+          return (
+            item.username.toLowerCase().includes(value.toLowerCase()) ||
+            item.bio.toLowerCase().includes(value.toLowerCase())
+          );
+        } else if (item.username) {
+          return item.username.toLowerCase().includes(value.toLowerCase());
+        } else if (item.description) {
+          return item.bio.toLowerCase().includes(value.toLowerCase());
+        }
+      });
+      setUserListSearch(newList);
+    } else {
+      setUserListSearch(userList);
+    }
+  };
+  const searchPod = (value) => {
+    if (value) {
+      const newList = podList.filter((item) => {
+        if (item.name && item.description) {
+          return (
+            item.name.toLowerCase().includes(value.toLowerCase()) ||
+            item.description.toLowerCase().includes(value.toLowerCase())
+          );
+        } else if (item.name) {
+          return item.name.toLowerCase().includes(value.toLowerCase());
+        } else if (item.description) {
+          return item.description.toLowerCase().includes(value.toLowerCase());
+        }
+      });
+      setPodListSearch(newList);
+    } else {
+      setPodListSearch(podList);
+    }
+  };
+  const array_chunks = (array, chunk_size) =>
```

Comment:
can we switch to camelCase? array_chunks => arrayChunks

Also wondering if this function can be refactored ?

Reply:
done